scripts/cluster-install/config.yaml

```diff
@@ -0,0 +1,16 @@
+auth_url:
+username:
+password:
+region_name:
+tenant_name:
+image_id: 70de1e0f-2951-4eae-9a8f-05afd97cd036 #centos-7 image_id
+machine_username: centos
+flavor_id: 2 #m1.small
+gateway_net_id:
+network_id:
+key_name:
+key_file_path:
+using_load_balancer: true or false
```

Comment:
I'd suggest just setting this to true. That gives enough indication that it's a boolean for the purposes of this while still making the default actually a valid value.

Reply:
Done, thanks.